intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,79 @@
+/* global $ */
+$(document).ready(function() {
+    if (Object.keys(raw_data).length > 0) {
+        $("#members-chart").show();
+        var canvas = $("#members-chart")[0];
+        var parsed_data = [];
+        $.each(raw_data, function(k, v) {
+            parsed_data.push([Date.parse(k, "YYYY-MM-DD"), v]);
+        });
+        parsed_data = parsed_data.sort(function(a, b) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'parsed_data' is not in camel case.](https://www.codacy.com/app/tjcsl/ion/file/2623510841/issues/source?bid=3176160&fileBranchId=3299251#l10)